src/tests/conftest.py

```diff
@@ -23,7 +23,7 @@ def get_available_openshift_versions() -> List[str]:
 
     if override_version:
         if override_version == consts.OpenshiftVersion.MULTI_VERSION.value:
-            return sorted(available_versions)
+            return sorted(available_versions, key=lambda s: list(map(int, s.split("."))))
```

Comment:
```suggestion
            return sorted(available_versions, key=semver.VersionInfo.parse)
```

Reply:
`ValueError: 4.6 is not valid SemVer string`